chore(docs): Update async-routes.mdx (#22849)
expo/expo: docs/pages/router/static-rendering/async-routes.mdx

import { Terminal } from '~/ui/components/Snippet';
import Video from '~/components/plugins/Video';

> > This guide requires `expo@49.0.0-alpha.3` or higher and `metro@0.76.3` &mdahs; everything listed here is still experimental. To use these features, you may need to [use Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing).

Reviewer: ```suggestion
> This guide requires `expo@49.0.0-alpha.3` or higher and `metro@0.76.3` &mdash; everything listed here is still experimental. To use these features, you may need to [use Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing).
```

sorry there was typo in my suggestion 🙏

## Static rendering

The current static rendering system (React) doesn't support React Suspense, so async routes will be disabled on web when rendering static pages. This will be improved in the future.

## Setup

> Enable the feature by setting the `asyncRoutes` option to `development` in your **app.json**:

Reviewer: ```suggestion
Enable the feature by setting the `asyncRoutes` option to `development` in your [app config](/versions/latest/config/app/):
```

@EvanBacon, do we add this property in [app config schema](https://docs.expo.dev/versions/latest/config/app/)?

```json app.json
{
  "expo": {
    "plugins": [
      [
        "expo-router",
        {
          "origin": "https://acme.com",
          "asyncRoutes": "development"
        }
      ]
    ]
  }
}
```

> You can set platform-specific settings for `asyncRoutes` using an object, `ios`, `android`, `web`, `default`:

Reviewer: ```suggestion
You can set platform-specific settings (`default`, `android`, `ios` or `web`) for `asyncRoutes` using an object:
```
